### ISSUE TITLE
[BUG] Exctracted annotations missing #color attribute ahead of highlighted text

### ISSUE THREAD

**Issue by jbguberman** [melat0nin/zotero-roam-export#43]
In the screenshots, ahead of each line of highlighted text, there's a "#Green" attribute. I'm assuming this is related to the color used to highlight the text within a PDF. When I export a citation from Zotero and import into Roam, this information is missing. If I go into Zotero's config and set `extensions.zotfile.pdfExtraction.colorAnnotations` to `true`, the extracted notes (both in Zotero and once imported into roam) represent the highlight color as bolded text within parentheses (rather than as a hashtag). Is, during the export process, this color attribute from ZotFile supposed to be converted from a parenthetical to a hashtag? Or did ZotFile maybe change something on their end? Either way, it would be really nice to be able to duplicate the formatting shown in the screenshots, since there's no find/replace or multi-cursor editing in Roam currently. Thanks!

**Comment by jbguberman** [comment]
Not sure if there's a way to address this through Zotero to Roam Export, but I solved the problem by editing the Zotfile 5.0.16 source code and then building from source. In `pdfAnnotations.js` I edited a few characters in line 286 such that the line now reads: `anno.markup = "<span style='background-color:rgba(" + anno.color.join(',') + ",.25)'><strong>#" + color_category + "</strong> - " + anno.markup + "</span>";`. I also set `extensions.zotfile.pdfExtraction.colorAnnotations` to `true` after reinstalling ZotFile. Now my extracted and exported annotations match the appearance of those in the screenshots of the Zotero to Roam Export readme.

**Comment by jbguberman** [comment]
Not sure if there's a way to address this through Zotero to Roam Export, but I solved the problem by editing the Zotfile 5.0.16 source code and then building from source. In `pdfAnnotations.js` I edited a few characters in line 286 such that the line now reads: `anno.markup = "<span style='background-color:rgba(" + anno.color.join(',') + ",.25)'><strong>#" + color_category + "</strong> - " + anno.markup + "</span>";`. I also set `extensions.zotfile.pdfExtraction.colorAnnotations` to `true` after reinstalling ZotFile. Now my extracted and exported annotations match the appearance of those in the screenshots of the Zotero to Roam Export readme.
